lib/lbt/bundle/Builder.js

```diff
@@ -542,15 +547,41 @@ class BundleBuilder {
 		});
 	}
 
-	writeBundleInfos(sections) {
+	logStringBundlingError(moduleName) {
+		log.error(
+			"Module " + moduleName + " requires top level scope and can only be embedded as a string " +
+			"(requires 'eval'), which is not supported with specVersion 4.0 and higher");
```

Comment:
Should we also give a hint to check our documentation to learn how to resolve this?
The warning message was always a bit confusing as it's not straight-forward to understand the scenario and reasoning behind, so I think it would help many users to get a link or hint to look into our documentation to learn what to do in this case (see https://github.com/SAP/ui5-tooling/pull/975)

Reply:
added the link to the future doccumenation